bcwallet/bcwallet.py

```diff
@@ -1120,6 +1121,8 @@ def wallet_home(wallet_obj):
 
     # Go to home screen
     while True:
+        new = 2
+        wallet_url = 'https://live.blockcypher.com/%s/xpub/%s/?subchain-indicies=0-1' % (coin_symbol_from_mkey(mpub), mpub)
```

Comment:
There's already a function called `get_public_wallet_url` that will do this for you. Is there a reason you didn't use it?